MatrixSDK/Data/RoomList/Common/MXBreadcrumbsRoomListDataFetcher.swift

```diff
@@ -91,7 +91,7 @@ internal class MXBreadcrumbsRoomListDataFetcher: NSObject, MXRoomListDataFetcher
         
         if let query = fetchOptions.filterOptions.query?.lowercased(), !query.isEmpty {
             recentRoomIds = recentRoomIds.filter({ roomId in
-                guard let displayname = session?.roomSummary(withRoomId: roomId).displayname else {
+                guard let displayname = session?.roomSummary(withRoomId: roomId)?.displayname else {
```

Comment:
I would add a comment at the top this line to remind that the function `roomSummary(withRoomId: )` might return a nil value.
In this way the issue should not happen again.